WikiWatershed/model-my-watershed
# Fix ARA on Edge Cases

## Overview

Better handles ARA requests for out of bounds queries, by using an updated version of the geoprocessing service which treats out of bound requests as NODATA tiles.

Connects #3239 

### Demo

![image](https://user-images.githubusercontent.com/1430060/73289101-1af9d000-41ca-11ea-8b66-126c5cee612c.png)

### Notes

Can't be merged until https://github.com/WikiWatershed/mmw-geoprocessing/pull/90 has been merged and a `4.0.2` release created and published for the geoprocessing service.

## Testing Instructions

* Check out this branch
* Reprovision the worker:

      $ vagrant reload worker --provision

* Test Roanoke Rapids HUC-8
  - [x] Ensure ARA results have values
* Test City of Philadelphia / Schuylkill HUC-12
  - [x] Ensure ARA results match `develop` / staging
* Test City of Phoenix-Salt River HUC-12
  - [x] Ensure ARA results match `develop` / staging
